MapboxNavigation/PollyVoiceController.swift

```diff
@@ -36,12 +37,22 @@ public class PollyVoiceController: RouteVoiceController {
     
     public override func alertLevelDidChange(notification: NSNotification) {
         guard shouldSpeak(for: notification) == true else { return }
-        speak(speechString(notification: notification, markUpWithSSML: true), error: nil)
+        
+        guard let routeProgress = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationRouteProgressKey] as? RouteProgress else {
+            assert(false)
+            return
+        }
+        
+        guard let userDistance = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationDistanceToEndOfManeuverKey] as? CLLocationDistance else {
+            assert(false)
+            return
+        }
+        speak(instruction: createSpeakText(from: routeProgress, at: userDistance, markUpWithSSML: true), error: nil)
         startAnnouncementTimer()
     }
     
-    override func speak(_ text: String, error: String?) {
-        assert(!text.isEmpty)
+    override public func speak(instruction: String, error: String?) {
```

Comment:
It’s fine to use `_ instruction` here, because `instruction` is precisely what’s being spoken.